apps/web/pages/event-types/[type].tsx

```diff
@@ -262,6 +264,26 @@ const EventTypePage = (props: inferSSRProps<typeof getServerSideProps>) => {
 
   const [requirePayment, setRequirePayment] = useState(eventType.price > 0);
   const [advancedSettingsVisible, setAdvancedSettingsVisible] = useState(false);
+  const [recurringEventDefined, setRecurringEventDefined] = useState(
+    eventType.recurringEvent && eventType.recurringEvent.count !== undefined
+  );
+  const [recurringEventInterval, setRecurringEventInterval] = useState(
+    (eventType.recurringEvent && eventType.recurringEvent.interval) || 1
+  );
+  const [recurringEventFrequency, setRecurringEventFrequency] = useState(
+    (eventType.recurringEvent && eventType.recurringEvent.freq) || RRuleFrequency.WEEKLY
+  );
+  const [recurringEventCount, setRecurringEventCount] = useState(
+    (eventType.recurringEvent && eventType.recurringEvent.count) || 12
+  );
```

Comment:
Same here about being able to track their values to dynamically change UI elements around plural mainly